### DISCUSSION TITLE
Elementor Forms: Brevo integration

### DISCUSSION THREAD

**Discussion by KhademOHAli1** [elementor/elementor#25782] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Brevo (sendinblue) Intgration with elmentor forms is not possible and requires extra plugin.

### Describe the solution you'd like.

Possible to register new newsletter subscribers for Brevo through elementor form feature

### Describe alternatives you've considered.

At the moment it is only possible through MailOptin and it is a very messy process. specially when it comes to Double Optin process.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
This is a long standing issue with many previous reports. A helpful dev has made a super light weight plugin though that adds this functionality. Works way better than mail opt in plugin and is tiny.

https://wordpress.org/plugins/integration-for-elementor-forms-sendinblue

P.S it is Brevo as Sendinblue is the pre rebrand name.

**Reply by Webtica** [reply]
Thanks for the kind words! :-)
